CHANGELOG.md

```diff
@@ -1,5 +1,61 @@
 # Changelog
 
+## 6.1.0
+
+Teleport 6.1 contains multiple new features, improvements, and bug fixes.
+
+## New Features
+
+### U2F for Kubernetes and SSH sessions
+
+Adds an option to authorize with 2nd factor when connecting to a node/k8s cluster.
+
+[RFD 14](https://github.com/gravitational/teleport/blob/master/rfd/0014-session-2FA.md)
+[RFD 15](https://github.com/gravitational/teleport/blob/master/rfd/0015-2fa-management.md)
+
+### Dual Authorization Workflows
+
+Request multiple users to review and approve access requests. Find out more in issue 5007.
+
+[#5071](https://github.com/gravitational/teleport/pull/5071)
```

Comment:
@fspmarshall  Do you have a short introduction and docs links that would go well in the changelog?